doc/source/conf.py

```diff
@@ -7,7 +7,7 @@
 # https://www.sphinx-doc.org/en/master/usage/configuration.html#project-information
 
 project = 'hotsos'
-copyright = '2023, edward.hope-morley@canonical.com'
+project_copyright = '2023, edward.hope-morley@canonical.com'
```

Comment:
The documentation says this is simply an alias to the `copyright` option. What is the advantage of changing it?

https://www.sphinx-doc.org/en/master/usage/configuration.html#confval-project_copyright

Reply:
Makes linters happy (Python's has [copyright](https://docs.python.org/3/library/constants.html#copyright))

Reply:
Ah, ok, I agree, that's a good thing :)